### PR TITLE
Namespace docs

### PR DESCRIPTION
**Pull request by mbwhite**
This is a close of the issue #3126

### CODE REVIEW

**Comment by mbwhite** [comment]
image
Well it can build!!

File: packages/composer-common/lib/system/historian.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Historian
+
```
Comment (nklincoln) [inline packages/composer-common/lib/system/historian.md]: 😕 mainly, er ... nope to this file ... blank or ipsum lorem pls!

File: packages/composer-documentation/test/testfile.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a test markdown file
```
Comment (nklincoln) [inline packages/composer-documentation/test/testfile.md]: called by?

File: packages/composer-tests-integration/features/cli-generate.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@cli @cli-generate
+Feature: Cli generate steps
+
+    Scenario: Using the CLI, I can update the network to a newer version
```
Comment (nklincoln) [inline packages/composer-tests-integration/features/cli-generate.feature]: this isn't the scenario name ur looking for 🤖 

Good test tho 👍

File: packages/composer-tests-integration/package.json [hunk 4]
```diff
@@ -15,9 +15,10 @@
     "test": "exit 0",
     "test-tagged": "cucumber-js --tags @busnet",
     "test-inner": "cucumber-js --tags 'not @cli-report'",
-    "test-inner-nohsm": "cucumber-js --tags 'not @hsm'",
-    "int-test": "npm run pretest && npm run lint && npm run start_verdaccio && node ./scripts/npm_serve && npm run test-inner && npm run stop_verdaccio",
-    "int-test-nohsm": "npm run pretest && npm run lint && npm run start_verdaccio && node ./scripts/npm_serve && npm run test-inner-nohsm && npm run stop_verdaccio"
+    "test-inner-nohsm": "cucumber-js --tags 'not @cli-hsm'",
+    "setup":"npm run pretest && npm run lint && npm run start_verdaccio && node ./scripts/npm_serve",
```
Comment (nklincoln) [inline packages/composer-tests-integration/package.json]: 👍

File: packages/composer-common/lib/system/historian.md (hunk 1 above)
Comment (mbwhite) [reply packages/composer-common/lib/system/historian.md]: gone

File: packages/composer-documentation/test/testfile.md (hunk 2 above)
Comment (mbwhite) [reply packages/composer-documentation/test/testfile.md]: defined within the bna file of the test network

File: packages/composer-tests-integration/features/cli-generate.feature (hunk 3 above)
Comment (mbwhite) [reply packages/composer-tests-integration/features/cli-generate.feature]: done

File: packages/composer-documentation/.eslintrc.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    strict: error
+    no-var: error
+    dot-notation: error
+    no-tabs: error
+    no-trailing-spaces: error
+    no-use-before-define: error
+    no-useless-call: error
+    no-with: error
+    operator-linebreak: error
+#    require-jsdoc:
```
Comment (sstone1) [inline packages/composer-documentation/.eslintrc.yml]: This needs uncommenting as per other modules - all the eslint configuration should be the same.

File: packages/composer-admin/composer-admin [hunk 6]
```diff
@@ -0,0 +1 @@
+composer-admin
```
Comment (sstone1) [inline packages/composer-admin/composer-admin]: wot is this?

File: packages/composer-cli/bin/composer [hunk 7]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```
Comment (sstone1) [inline packages/composer-cli/bin/composer]: wot is this? isn't this cli.js?

**Comment by mbwhite** [comment]
@sstone1 errant file have gone

**Pull request merged**